addons/info/src/components/types/ObjectOf.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+import PrettyPropType from './PrettyPropType';
+import { TypeInfo } from './proptypes';
+
+const ObjectOf = ({ propType }) =>
+  <span>
```

Comment:
This is a lot of spans, some are even empty spans, can you explain why this is?

Reply:
This is cleaned up, but github isn't picking it up because only the below lines were changed.

Reply:
✅